### ISSUE TITLE
Phunk's process is not ended when closing the application.

### ISSUE THREAD

**Issue by T-Lacerte** [shibadevs/phunk#3]
# Problem
Phunk's process doesn't seem to end when the application is closed. If phunk is opened, for example, 3 times, 3 processes will be left in the background.
# Repro steps
1. Open the application
2. Close the application with the window's cross
3. Open Windows's task manager
4. Search for «Phunk» and check if the process is present.

**Comment by shibadevs** [comment]
I will get into the bottom of this. I'm using Adonis UI which is responsible for the UI that has its own window services. So sadly, I don't know why it isn't closing properly, but I'll be checking it.

**Comment by shibadevs** [comment]
Bug has been fixed on the latest version of Phunk (1.0.3)

https://github.com/shibadevs/phunk/blob/main/Phunk/MainWindow.xaml.cs
```csharp
// Fix for Phunk not closing when process has started
        private void AdonisWindow_Closing(object sender, System.ComponentModel.CancelEventArgs e)
        {
            Environment.Exit(Environment.ExitCode);
        }
```
